user/models.py

```diff
@@ -0,0 +1,75 @@
+from django.db import models
+
+class User(models.Model):
+    name                = models.CharField(max_length=50)
+    nick_name           = models.CharField(max_length=50)
+    email               = models.EmailField(max_length=100)
+    password            = models.CharField(max_length=200)
+    phone_number        = models.CharField(max_length=50)
+    creator_flag        = models.BooleanField(default=False, null=True)
```

Comment:
```suggestion
    is_creator        = models.BooleanField(default=False, null=True)
```
boolean 필드로 특정한 상태가 맞는지 아닌지를 구별하실 때에는 `is_` 또는 `has_`를 활용하는 습관을 들이시는 것이 좋습니다.
의미에 맞다면요 :)
이 의미가 맞을까요?

Reply:
넵, 맞습니다. 수정하였습니다!